builtInFunctions/esdtRoles.go

```diff
@@ -76,6 +87,15 @@ func (e *esdtRoles) ProcessBuiltinFunction(
 	return vmOutput, nil
 }
 
+// Nonces on multi shard NFT create ar from (LastByte * MaxUint64 / 256), this is in order to differentiate them
+// even like this, if one contract makes 1000 NFT create on each block, it would need 14 million years to occupy the whole space
+// 2 ^ 64 / 256 / 1000 / 14400 / 365 ~= 14 million
+func computeStartNonce(destAddress []byte) uint64 {
```

Comment:
this function will return only 256 possible values. Is this ok? I think we should have added a current nonce to the computed startNonce

Reply:
This is OK. this is the startNonce to be computed per lastAddress when multishardNFTCreate role is set.